python/src/main/java/org/apache/zeppelin/python/PythonInterpreter.java

```diff
@@ -116,7 +115,6 @@ public void open() throws InterpreterException {
     }
 
     try {
-      this.usePy4jAuth = Boolean.parseBoolean(getProperty("zeppelin.py4j.useAuth", "true"));
```

Comment:
This change is based on the fact that `isSecretSupported` is always true for Spark 3.2 and above.

property `zeppelin.py4j.useAuth` is only used internally, and is not exposed to the end user, the removal should be safe.

Reply:
:heavy_check_mark: